buildtest/menu/build.py

```diff
@@ -111,6 +149,35 @@ def func_build_subcmd(args):
     # Discover list of one or more config files based on path provided
     config_files = discover_configs(args.config)
 
+    # if no files discovered let's stop now
+    if not config_files:
+        msg = "There are no config files to process."
+        logger.error(msg)
+        sys.exit(msg)
+
+    print("\n")
+    print("{:^45}".format("Discovered Files"))
+    print("\n")
```

Comment:
Can we put the newlines with other statements so this single block doesn't need to take up 7 lines?

Reply:
Makes sense i can make this adjustment